### PR TITLE
Clima master

### PR DESCRIPTION
**Pull request by yairchn**
Adding RH to the plots. 
The detrainment is now based on the relative humidity difference so I added the comparison of RH to the plotting and related the mixture buoyancy with RH in the closures plots (b_mix is not used now)

### CODE REVIEW

**Comment by yairchn** [comment]
This requires - yet again - to delete the les files that will be downloaded as the LES files before had no RH in them

File: tests/les_data/reduce_pycles_netcdf.py [hunk 1]
```diff
@@ -36,7 +36,10 @@ def main():
 
     updraft_ddz_p_alpha_ = data.groups['profiles'].variables['updraft_ddz_p_alpha']
     rho_ = data.groups['reference'].variables['rho0_half']
-    p0_ = data.groups['reference'].variables['rho0_half']
+    p0_ = data.groups['reference'].variables['p0_half']
+
+    t = np.multiply(data.groups['profiles'].variables['t'], 1.0)
```
Comment (trontrytel) [inline tests/les_data/reduce_pycles_netcdf.py]: How about t=np.array(data.groups['profiles'].variables['t']) ?

The same for z

Comment (yairchn) [reply tests/les_data/reduce_pycles_netcdf.py]: sure

File: tests/les_data/reduce_pycles_netcdf.py [hunk 2]
```diff
@@ -345,6 +370,14 @@ def calc_third_m(var, var2, var3, covar, upd_frac):
     third_m = np.subtract(var3,C)
     return third_m
 
+def relative_humidity(p0, qt, qv, T):
```
Comment (trontrytel) [inline tests/les_data/reduce_pycles_netcdf.py]: Could you use the functions already defined in SCAMPy?

File: tests/plots/plot_scripts.py [hunk 3]
```diff
@@ -123,11 +125,11 @@ def plot_closures(scm_data, les_data, tmin, tmax, title, folder="plots/output/")
                            scm_data["updraft_area"][:, t0_scm : t1_scm], axis=1\
                           ) / scm_data["rho_half"][:],\
                 np.nanmean(scm_data["turbulent_entrainment"][:, t0_scm : t1_scm], axis=1),\
-                np.nanmean(scm_data["updraft_buoyancy"][:, t0_scm : t1_scm], axis=1),\
+                np.nanmean(scm_data["updraft_RH"][:, t0_scm : t1_scm], axis=1),\
                 np.nanmean(scm_data["entrainment_sc"][:, t0_scm : t1_scm], axis=1)]
 
     x_lab = ["eddy_diffusivity", "mixing_length [km]", "non hydro pressure [Pa]",\
-             "turbulent_entrainment", "buoyancy [m/s^2]", "entr and detr [1/m]"]
+             "turbulent_entrainment", "RH [m/s^2]", "entr and detr [1/m]"]
```
Comment (trontrytel) [inline tests/plots/plot_scripts.py]: wrong units

Comment (yairchn) [reply tests/plots/plot_scripts.py]: good catch

**Pull request closed without merging**